vmm/src/device_manager.rs

```diff
@@ -895,6 +984,14 @@ impl DeviceManager {
         Ok(device_manager)
     }
 
+    pub fn get_serial_pty(&self) -> Option<&Arc<Mutex<(File, File)>>> {
```

Comment:
I thought it was Rust style to omit "get_" prefix in accessors?

Reply:
Fixed